### PR TITLE
Feature/backend scripts - Added scripts to read and write to database as well as a script to add data into the database.

### PR DESCRIPTION
**Pull request by RubberDuckDestroyer**

### CODE REVIEW

File: lib/controllers/loginController.dart [hunk 1]
```diff
@@ -65,4 +67,23 @@ class LoginController {
     final result = await resetPasswordFor(this._db, email, password);
     return result;
   }
+
```
Comment (Prashar-star) [inline lib/controllers/loginController.dart:70]: Logic is looking good.

File: lib/scripts/readValue.dart [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    }
+  } catch (e) {
+    print("There was an error when trying to read from collection " +
+        coll +
+        " : ");
+    print(e);
+  }
+
+  return result;
+}
```
Comment (njosling) [inline lib/scripts/readValue.dart:25]: Excellent coding standards

File: lib/scripts/createMeal.dart [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import 'package:fitness_freaks/models/postTypes.dart';
```
Comment (Prashar-star) [inline lib/scripts/createMeal.dart:1]: The script's looking good.  Love the way you put in a futuristic thought to it by adding youtube link etc.

File: lib/scripts/readValue.dart [hunk 4]
```diff
@@ -0,0 +1,25 @@
+/*
```
Comment (Prashar-star) [inline lib/scripts/readValue.dart:1]: Good work on error handling there.

File: test/writeValue_test.dart [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+      final didWrite = await controller.writeValue(collection, document);
+
+      expect(didWrite, equals(true));
+    });
+  } catch (e) {
+    print("There was an error.");
+    print(e);
+  }
+}
```
Comment (njosling) [inline test/writeValue_test.dart:29]: Variables have relevant and logical names, whitespace is excellent

File: lib/scripts/writeVaue.dart [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+    }
+  } catch (e) {
+    print("There was an error when trying to write to collection " +
+        coll +
+        " : ");
+    print(e);
+  }
+
+  return result;
+}
```
Comment (njosling) [inline lib/scripts/writeVaue.dart:24]: Logic is perfect, a few comments could be added

**Review by njosling — APPROVED** [review]

**Review by Prashar-star — APPROVED** [review]
Overall, looking good. Code is easy to read and pretty self-explanatory

**Pull request merged**